portfolio/src/main/webapp/script.js

```diff
@@ -38,13 +38,13 @@ function randomNumber() {
  */
 function secretPage(){
   //Generates a number between 1 and 200 for both numbers.
-  var a = randomNumber();
-  var b = randomNumber();
+  let a = randomNumber(200,1);
+  let b = randomNumber(200,1);
```

Comment:
Per the Google Javascript style guide: https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace

> Horizontal Whitespace is required:
> After a comma (,) or semicolon (;). Note that spaces are never allowed before these characters.

```suggestion
  let a = randomNumber(200, 1);
  let b = randomNumber(200, 1);
```